File: src/SamplePublishing/RA.SamplesForDocumentation/SampleModels/BaseModel.cs

using System.Collections.Generic;

using RA.Models.Input;

namespace RA.SamplesForDocumentation.SampleModels
{
	public class BaseModel
	{
		public string Name { get; set; }
		public string Description { get; set; }
		public string SubjectWebpage { get; set; }
		public string CTID { get; set; }
		public string DateEffective { get; set; }

		public List<Place> Address { get; set; } = new List<Place>();
	}
	
}

File: src/SamplePublishing/VisualBasicPublishingSamples/SampleModels/BaseModel.vb
Imports RA.Models.input

'Namespace VisualBasicSamplePublishing
Public Class BaseModel
	Public Property Name As String
	Public Property Description As String
	Public Property SubjectWebpage As String
	Public Property Ctid As String
	Public Property DateEffective As String
	Public Property Address As List(Of Place) = New List(Of Place)
End Class
'End Namespace
